js/datecalc.js

```diff
@@ -0,0 +1,18 @@
+import { DateTime } from './luxon.min.js';

```

Comment:
здесь надо подключать установленный через npm luxon
вот так
```suggestion
import { DateTime } from 'luxon';

```